cxxfilt/__init__.py

```diff
@@ -20,8 +20,7 @@ class InternalError(Error):
 
 
 class InvalidName(Error):
-    '''Exception raised when:
-    mangled_name is not a valid name under the C++ ABI mangling rules.'''
+    '''When mangled name is not a valid name under the C++ ABI mangling rules.'''
```

Comment:
Suggest changing it to `mangled_name is not a valid name under the C++ ABI mangling rules`

`mangled_name` is an attribute of the Exception.

Reply:
I think it should stay that way, because `mangled_name` has no context in the body of the exception. (It has a meaning in the `__init__` method below, so `mangled_name` cam be used there)

Reply:
`mangled_name` is used pretty consistently in the code so I don't think it lost meaning, but I'm fine with changing that. I believe "When" should be removed though.

Reply:
Revamped check new version